pymeasure/instruments/hp/hp33120A.py

```
@@ -44,6 +44,8 @@ def __init__(self, adapter, name="Hewlett Packard 33120A Function Generator", **
         )
         self.amplitude_units = 'Vpp'
 
+    _init_comm_pairs = [(b'SOUR:VOLT:UNIT VPP', None)]
```

Ruff (0.15.12)

[warning] Mutable default value for class attribute

(RUF012)